build.gradle

```diff
@@ -18,20 +18,85 @@ plugins {
 	id 'maven-publish'
 }
 
-javadoc {
-	options.encoding = 'UTF-8'
-	//remove this to see all the missing tags/parameters.
-	options.addStringOption('Xdoclint:none', '-quiet')
-}
+allprojects {
```

Comment:
no matter what I think the changes to enable additional modules in this repo should stay. enable us to split up or add things where it makes the most sense.

Reply:
Any changes I made to the gradle build file are there only to get things to work, which was already difficult enough. Very likely I made mistakes or did it in an improper way.